src/app/dashboard-widgets/applications-widget/applications-widget.component.html

```diff
@@ -1,41 +1,58 @@
 <f8-feature-toggle featureName="ApplicationsCard">
   <div class="applications-widget card-pf f8-card" user-level>
     <div class="card-pf-heading f8-card-heading">
+      <div class="card-pf-heading-details"
+           *ngIf="applicationsInProgress > 0 && (stageConfigsAvailable || runConfigsAvailable)">
+        <p>
+          Setting up {{applicationsInProgress}} applications.
+          <a [routerLink]="['/', loggedInUser?.attributes?.username, this.currentSpace?.attributes?.name, 'create', 'pipelines']">View progress.</a>
```

Comment:
Maybe this should be checked with UXD, but I think the period should be dropped.

`View progress` instead of `View progress.` This follows the other widgets and linked text in the UI.

Reply:
Considering the first sentence "Setting up {{applicationsInProgress}} applications." uses a period, then the next sentence should have one as well.

Reply:
I'm not going to argue over a period -- reviews shouldn't be painful.

Feel free to obtain clarification from UXD.